services/brig/test/integration/API/Metrics.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TupleSections       #-}
+{-# LANGUAGE ViewPatterns        #-}
+
+-- | See also: services/spar/test-integration/Test/Spar/APISpec.hs
+module API.Metrics (tests) where
+
+import Imports
+import Bilge
+import Brig.Types.User
+import Control.Lens
+import Data.ByteString.Conversion
+import Data.String.Conversions (cs)
+import Data.Aeson
+import Data.Aeson.Lens
+import Data.Set as Set
+import Test.Tasty
+import Test.Tasty.HUnit
+import Util
+
+
+tests :: Manager -> Brig -> IO TestTree
+tests manager brig = do
+    return $ testGroup "metrics"
+        [ testCase "work" . void $ runHttpT manager (testMetricsWaiRoute brig)
+        ]
+
+-- | path parsing works; http verb, captured variables are not distinguished in the metrics.
```

Comment:
I'm very uneasy about this: Does the change introduced in this PR lead to *less* precise metrics then before? If so, why are you doing this?

Example of how metrics currently (before this PR) work on `/i/monitoring` for brig. You can see that metrics are separated by route and verb:

```json
{
   "net" : {
      "connections" : 6,
      "resources" : {
(....)
         "/properties/:key" : {
            "GET" : {
               "time" : {
                  "240" : 0,
                  "339" : 0,
                  "170" : 1,
                  "679" : 0,
                  "0" : 1781,
                  "480" : 0,
                  "60" : 11,
                  "85" : 6,
                  "960" : 0,
                  "1358" : 0,
                  "120" : 0,
                  "30" : 11,
                  "42" : 0
               },
               "status" : {
                  "200" : 533,
                  "404" : 1277
               }
            },
            "PUT" : {
               "time" : {
                  "240" : 0,
                  "339" : 0,
                  "170" : 5,
                  "679" : 0,
                  "480" : 0,
                  "0" : 911,
                  "85" : 7,
                  "60" : 2,
                  "1358" : 0,
                  "960" : 0,
                  "42" : 0,
                  "30" : 6,
                  "120" : 2
               },
               "status" : {
                  "200" : 933
               }
            }
         },
(....)
}
```

Reply:
Sorry, i was wrong about this: a28849d